packages/plugin-core/src/components/views/NoteGraphViewFactory.ts

```diff
@@ -136,6 +143,10 @@ export class NoteGraphPanelFactory {
         if (this._onEngineNoteStateChangedDisposable) {
           this._onEngineNoteStateChangedDisposable.dispose();
         }
+        const profile = getDurationMilliseconds(start);
+        AnalyticsUtils.track(DENDRON_COMMANDS.SHOW_NOTE_GRAPH.key, {
+          timespan: profile,
```

Comment:
we already have a precendent for using [duration](https://airtable.com/appVUixSC6jZbSUI1/tblHzTJJL5aVhJMYq/viwgxTbxGhjlLuBSh/recxnNkjek9LvLRZ5?blocks=hide) to measure profile information. lets use that

Reply:
Hmm..I was actually trying to measure how long did user had the graph view open.